databases/postgresql15/Portfile

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup select 1.0
+PortGroup compilers 1.0
+PortGroup compiler_blacklist_versions 1.0
+PortGroup muniversal 1.0
+
+#remember to update the -doc and -server as well
+name                postgresql15
+version             15.0
+revision            1
```

Comment:
drop this line? i think it should be the implicit revision 0

Reply:
Fixed.